contrib/guix/libexec/build.sh

```diff
@@ -199,17 +245,22 @@ mkdir -p "$DISTSRC"
                     --disable-maintainer-mode \
                     --disable-dependency-tracking \
                     ${CONFIGFLAGS} \
-                    CFLAGS="${HOST_CFLAGS}" \
-                    CXXFLAGS="${HOST_CXXFLAGS}" \
+                    ${HOST_CFLAGS:+CFLAGS="${HOST_CFLAGS}"} \
+                    ${HOST_CXXFLAGS:+CXXFLAGS="${HOST_CXXFLAGS}"} \
                     ${HOST_LDFLAGS:+LDFLAGS="${HOST_LDFLAGS}"}
 
     sed -i.old 's/-lstdc++ //g' config.status libtool src/univalue/config.status src/univalue/libtool
 
     # Build Bitcoin Core
     make --jobs="$MAX_JOBS" ${V:+V=1}
 
-    # Perform basic ELF security checks on a series of executables.
-    make -C src --jobs=1 check-security ${V:+V=1}
+    case "$HOST" in
+        *darwin*) ;;
```

Comment:
Originally I skipped them because we don't perform them in gitian osx builds, which I thought had some significance. As of 05a31a95832b592ee565d3a429afcf6cbf5ae672 they are enabled for all builds in Guix. Sweet!